portafly/src/components/pages/accounts/listing/AccountsListingTable.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect } from 'react'
```

Comment:
Not entirely convinced about the path and nomenclature `listing`, maybe this could be `list` or `collection`. This applies to all of the paths and name of files

Reply:
I strongly suggest we cope with that in another PR, perhaps of #1941

Reply:
#1960